public/js/common.js

```diff
@@ -6,12 +6,18 @@ const LRR = {};
 /**
  * Quick 'n dirty HTML encoding function.
  * @param {*} r The HTML to encode
+ * @param {boolean} isTimestamp Option to convert timestamp to date string
  * @returns Encoded string
  */
-LRR.encodeHTML = function (r) {
+LRR.encodeHTML = function (r, isTimestamp = false) {
     if (r === undefined) return r;
-    if (Array.isArray(r)) return r[0].replace(/[\x26\x0A\<>'"]/g, (r2) => `&#${r2.charCodeAt(0)};`);
-    else return r.replace(/[\x26\x0A\<>'"]/g, (r2) => `&#${r2.charCodeAt(0)};`);
+    if (isTimestamp === true) return (new Date(r * 1000)).toLocaleDateString();
+    if (Array.isArray(r)) {
+        return r[0].replace(/[\x26\x0A<>'"]/g, (r2) => `&#${r2.charCodeAt(0)};`);
```

Comment:
Comes from https://stackoverflow.com/a/31638198 .

Reply:
Have a look at my following code, does it solve the same problem as the "[escape HTML](https://stackoverflow.com/questions/3043775/how-to-escape-html)" asked?
![image](https://user-images.githubusercontent.com/10560038/165760719-30f0fdd1-9aab-499f-931d-b6fbd188ad16.png)

Reply:
iirc this approach doesn't escape `"` , so it might cause issues 👎 The function is mostly used to display titles, which often contain quotes.

![image](https://user-images.githubusercontent.com/8237712/165767434-7f290f2f-5fb1-475b-ac10-d5dc129be22a.png)

Reply:
Is there any real-world examples about it? I don't think leaving " in HTML may cause some problem.
Also, afaik, encoding is used to prevent XSS attack, as well as preventing characters like `<` and `>` being parsed by the browser.
I don't think leaving `"` can cause any problem. Some real test cases can be more convincing.

Reply:
Nevermind. I found it was part of ESLint's problem
I changed `\x26\x0A` into `\n&`, then the ESLint just shut up.
![image](https://user-images.githubusercontent.com/10560038/165777812-af52263a-c62f-40ba-9b0a-6be59977abda.png)